#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_tries_option.dpatch by Raimund Specht <raimund@spemaus.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix the --tries commandline option to work again

@DPATCH@
--- cryptsetup/lib/setup.c	2007-03-07 17:08:52.000000000 +0100
+++ cryptsetup/lib/setup.c	2007-03-07 17:08:32.000000000 +0100
@@ -639,7 +639,7 @@
 	if(!password) {
 		r = -EINVAL; goto out;
 	}
-	if((r = LUKS_open_any_key(options->device, password, passwordLen, &hdr, &mk, backend) < 0)) {
+	if((r = LUKS_open_any_key(options->device, password, passwordLen, &hdr, &mk, backend)) < 0) {
 		set_error("No key available with this passphrase.\n");
 		goto out1;
 	}
